bot/models/player.py

```diff
@@ -81,6 +81,17 @@ def add_letter(self):
                 return(f"{self.state['username']}, you can have an **{letter}**")
             return(f"{self.state['username']}, you can have a **{letter}**")
 
+    def add_vowel(self):
```

Comment:
I'd prefer to see this refactored into a variant of the `add_letter` method. So we'd have a method:
`add_letter(self, restricted_letters: list=[])` ... which by default would have no restricted letters.

Then `add_vowel` would call `add_letter`, and specify the vowels as restricted letters.

Also note that (IMHO?) `add_vowel` shouldn't create duplicates, so it should also respect the letters in hand. We can do that by appending the consonants to the letters in hand when requesting a letter.

Another approach -- which might be better? Would be to have a parameter on `add_letter`: e.g. "type" -- which would default to any, but could be restricted to vowel (or other).

Obvs I have opinions here! LMK if you'd like to chat about it, or alternatively, we push forward as-is and you can assign me an issue to do this refactor.

Reply:
I'm fine with any of those changes, and agree on duplicate prevention for add_vowel. Mind if I punt that follow-up CL to you?

Reply:
New issue created

Reply:
Cool thanks -- on it.